documentation/stringsGenerator/Sources/stringsGenerator/ParseTable.swift

```diff
@@ -30,8 +30,8 @@ class ParseTable {
 
         let sourceURL = URL(string: localizationSource)!
         //print("filepath given to script \(fileURL)")
-        guard let sourceFile = String(data: try? Data(contentsOf: sourceURL), encoding: .utf8) else {
-            fatalError("Should never fail, or something is really wrong.")
+        guard let contents = try? Data(contentsOf: sourceURL), let sourceFile = String(data: contents, encoding: .utf8) else {
+            fatalError("Should never fail, are files missing?")
```

Comment:
Perhaps we can interpolate `localizationSource` here to be more specific about which file may be missing?

Reply:
Yea, such an easy fix to make things a lot better.